shared/reducers/config.tsx

```diff
@@ -33,22 +33,6 @@ export default Container.makeReducer<Actions, Types.State>(Constants.initialStat
       ).username
     }
   },
-  [Tracker2Gen.updatedDetails]: (draftState, action) => {
```

Comment:
This comes in from Identify3 but we don't need it, following and followers should be kept up to date outside of the identify system.